packer_templates/sles/scripts/remove-dvd-source.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh -eux
 
 zypper removerepo `zypper repos | grep 'SLES' | awk '{ print $3 }' | grep "^SLES"`;
-zypper refresh
```

Comment:
I found this sped up usage of the images because the repo diff would be less between what we shipped and current.

Reply:
Well, keeping that line without at least adding the explicit 'exit 0' fails the build for me.